src/components/MovieDetailContainer.js

```diff
@@ -4,7 +4,7 @@ import "./MovieDetailContainer.css";
 function MovieDetailContainer({movie}) {
   const {title,tagline,budget,revenue,overview,runtime,genres} = movie
 
-  const genresToDisplay = genres.map(genre=> <p>{genre} </p>)
+  const genresToDisplay = genres.map((genre,index)=> <p key={index}>{genre} </p>)
```

Comment:
Indexes can be useful for using as a key, but keep in mind that they have caveats. Look into this section of the docs to learn more about when you don't want to use the index as a key: https://reactjs.org/docs/lists-and-keys.html#keys